src/hearings/containers/request-hearing/hearing-requirements/hearing-requirements.component.spec.ts

```diff
@@ -1,50 +1,197 @@
+        ''
+      ],
+      panelComposition: [
+        {
+          memberType: '',
+          count: 1
+        }
+      ],
+      judiciaryPreferences: [
+        {
+          memberID: '',
+          memberType: '',
+          requirementType: 'EXCLUDE'
+        }
+      ],
+      judiciarySpecialisms: [
+        ''
+      ]
+    },
+    hearingIsLinkedFlag: false,
+    parties: [
+      {
+        partyName: 'Jane and Smith',
+        partyChannel: '',
+        unavailability: [
+          {
+            start: '2021-12-10T09:00:00.000+0000',
+            end: '2021-12-31T09:00:00.000+0000'
+          }
+        ]
+      },
+      {
+        partyName: 'DWP',
+        partyChannel: '',
+        unavailability: [
+          {
+            start: '2021-12-20T09:00:00.000+0000',
+            end: '2021-12-31T09:00:00.000+0000'
+          }
+        ]
+      }
+    ],
+    caseFlags: {
+      flags: [
+        {
+          partyName: 'Jane and Smith',
+          flagId: 'Language Interpreter',
+          flagDescription: 'Spanish interpreter required',
+          flagStatus: 'ACTIVE'
+        },
+        {
+          partyName: 'Jane and Smith',
+          flagId: 'Sign language interpreter',
+          flagDescription: 'Sign language interpreter required',
+          flagStatus: 'ACTIVE'
+        },
+        {
+          partyName: 'Jane and Smith',
+          flagId: 'Reasonable adjustment',
+          flagDescription: 'Hearing loop required',
+          flagStatus: 'ACTIVE'
+        },
+        {
+          partyName: 'DWP',
+          flagId: 'case flag 1',
+          flagDescription: 'case flag 1 description',
+          flagStatus: 'ACTIVE'
+        }
+      ],
+      flagAmendURL: '/'
+    },
+  } as ServiceHearingValuesModel;
 
   const initialState = {
     hearings: {
       hearingList: {
         caseHearingMainModel: [
           {
-            hmctsServiceID: 'SSCS'
+            hmctsServiceID: 'TEST'
           }
         ]
       },
-      hearingValues: null,
-      hearingRequest: null,
-      hearingConditions: null,
+        hearingValues:  {
+          serviceHearingValuesModel: hearingValueModel,
+          lastError: null,
+      },
     }
   };
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      declarations: [HearingRequirementsComponent],
+      imports: [],
+      declarations: [ HearingRequirementsComponent, MockHearingPartiesComponent ],
       providers: [
         provideMockStore({initialState}),
-        {provide: HearingsService, useValue: hearingsService},
-      ]
-    }).compileComponents();
+      ],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    })
+    .compileComponents();
+  });
 
+  beforeEach(() => {
     fixture = TestBed.createComponent(HearingRequirementsComponent);
     component = fixture.componentInstance;
     fixture.detectChanges();
+    spyOn(component, 'convertMapToArray').and.callThrough();
+    spyOn(component, 'assignHearingValue').and.callThrough();
```

Comment:
This will cause the fake unit tests coverage rate.
These two spys actually asked the tests to skip the two main method.